jbt-it/neomdb
# #309 workshops overview

How to test:

- go to /workshops and check if everything is displayed correctly in the mobile and desktop version
- open the add workshop dialog and check if it works, it should activate an alert when saving which will be replaced by actually adding it to the database when the backend is fixed
